addons/syntaxhighlighter/script.js

```diff
@@ -67,34 +67,46 @@ SyntaxHighlighter.all();
 
     $('body').on( 'submit', '[apcodeform]', function(e){
 
-      var code = $(this).find('textarea').val();
-      code = code.replace(/^\s\s*/, '').replace(/\s\s*$/, '');
-      code = tinyMCE.DOM.encode(code).replace(/^[\s]+/gm, function(m) {
-        var leadingSpaces = arguments[0].length;
-        var str = '';
-        while(leadingSpaces > 0) {
-          str += '&nbsp;';
-          leadingSpaces--;
-        }
-        return str;
-      });
-      code = code.replace(/[\r\n]\s*/g, '<br />');
-
-      var lang = $(this).find('select').val();
-      var tag = $(this).find('input').is(":checked") ? 'code' : 'pre';
-
-      var attr = 'language="'+lang+'"';
-      var cont = '[apcode '+attr+']<'+tag+' data-mce-contenteditable="false">'+code+'</'+tag+'>[/apcode]';
-      tinymce.activeEditor.insertContent(cont);
-      tinymce.activeEditor.focus();
-      tinymce.activeEditor.selection.collapse(0);
-      AnsPress.hideModal('code');
-
-      return false;
+		var code = $(this).find('textarea').val();
```

Comment:
@qstudio Can you please retain existing indentation. Thanks

Reply:
This is not our change, I'll note all the changes in the PR which we made.

Reply:
Please ignore then